devel/kube/linode/deployment.yml

```diff
@@ -20,14 +20,14 @@ spec:
             claimName: pvc-labbox-ephys-ephys1-b
       containers:
       - name: labbox-ephys-ephys1
-        image: magland/labbox-ephys:0.4.11
+        image: magland/labbox-ephys:0.4.12
         imagePullPolicy: Always
         env:
         - name: HOME
           value: /tmp
         - name: KACHERY_P2P_START_DAEMON_OPTS
-          value: --port 14207 --host xxxxxx
-        - name: LABBOX_CONFIG_URI
+          value: --port 14207 --websocket-port 14208 --label xxxxxx --host xxxxxx --config xxxxxx --noudp
+        - name: LABBOX_EPHYS_CONFIG
```

Comment:
--noudp option is now available on the latest kachery-p2p. This tells the daemon not to use UDP. I was having a lot of trouble exposing a UDP port on the kube cluster on linode. So I decided it's best to just let it revert to transfer via proxy server.